src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/SearchBar.xaml

```diff
@@ -24,62 +26,81 @@
 
     <!--  Back button and search box  -->
     <Grid
-        Margin="0,-10,0,0"
         Padding="0,12,0,12"
         HorizontalAlignment="Stretch"
         VerticalAlignment="Stretch">
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="44" />
             <ColumnDefinition Width="Auto" />
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
 
         <!--  Back button  -->
         <FontIcon
-            Margin="24,0,2,0"
-            HorizontalAlignment="Right"
+            Margin="14,0,0,0"
+            HorizontalAlignment="Center"
+            ui:VisualExtensions.NormalizedCenterPoint="0.5,0.5"
             AutomationProperties.AccessibilityView="Raw"
             Foreground="{ThemeResource TextFillColorSecondaryBrush}"
             Glyph="&#xE721;"
-            Visibility="{x:Bind Nested, Mode=OneWay, Converter={StaticResource ReverseBoolToVisibilityConverter}}" />
+            Visibility="{x:Bind CurrentPageViewModel.IsNested, Mode=OneWay, Converter={StaticResource ReverseBoolToVisibilityConverter}}">
```

Comment:
Now that the back button is enabled here, it's sucking up the focus that we're trying to direct to the searchbox as we navigate and such.

Should we move the Back button out of this control and into the shell page itself?